### ISSUE TITLE
semver mismatches for same git tag with train-141

### ISSUE THREAD

**Issue by jrgm** [mozilla/fxa#1731]
In the past, e.g., v1.140.3, if I deployed the same tag on all stacks, the `__version__` response of each would show the same git commit and semver. 

Now, only the content server shows v1.141.1 even though, they all show the same git commit. (Yeah, I see the bump of one semver in 67f46f7, but this is scripted, right?)

```
[
  {
    "commit": "67f46f72f9c77b5c06ebadc05f347ea498b9baec",
    "version": "1.141.1",
    "l10n": "d1a11742d5ff2b5186059bc46321f4c569de03a2",
    "tosPp": "3d4265cd522bf7a616cc3537750a96305d690bb3",
    "source": "https://github.com/mozilla/fxa"
  },
  {
    "version": "1.141.0",
    "commit": "67f46f72f9c77b5c06ebadc05f347ea498b9baec",
    "source": "https://github.com/mozilla/fxa"
  },
  {
    "version": "1.141.0",
    "commit": "67f46f72f9c77b5c06ebadc05f347ea498b9baec",
    "source": "https://github.com/mozilla/fxa"
  },
  {
    "version": "1.141.0",
    "commit": "67f46f72f9c77b5c06ebadc05f347ea498b9baec",
    "source": "https://github.com/mozilla/fxa"
  }
]
```

**Comment by philbooth** [comment]
Oh, well spotted, yeah. It means that future runs of the release script will fail to bump too because they do `sed "s/$OLD_VERSION/$NEW_VERSION/`, which won't match any more. I'll tag a `.2` to fix.

**Comment by philbooth** [comment]
Fixed in 141.2
